lib/src/value/color.ts

```diff
@@ -3,371 +3,926 @@
 // https://opensource.org/licenses/MIT.
 
 import {Value} from './index';
+import {valueError} from '../utils';
 import {
   fuzzyAssertInRange,
   fuzzyEquals,
+  fuzzyHashCode,
   fuzzyRound,
   positiveMod,
 } from './utils';
-import {hash} from 'immutable';
+import {List, hash} from 'immutable';
+import Color from 'colorjs.io/dist/color.legacy.cjs';
```

Comment:
@nex3 The default ColorJS build does not support node14 -- do we need to maintain support for that?